app/assets/stylesheets/notifications.scss

```diff
@@ -428,3 +431,134 @@
     @include load-more;
   }
 }
+
+
+.notifications-filter-menu__mobile {
+  display: none;
+
+  @media screen and (max-width: $breakpoint-m) {
+    display: block;
+  }
+}
+
+// Notifications filtering on mobile.
+.notifications-mobile-filters {
+  @media screen and (min-width: 950px) {
+    display: none;
+  }
+
+  display: none;
+
+  &.showing {
+    display: flex;
+  }
+
+  background:  var(--base-inverted);
+}
+
+.crayons-dropdown {
+
+  &__menu-overlay {
+    display: none;
+
+    &.showing {
+      display: flex;
+    }
+
+    &:before {
+      content: "";
+      position: fixed;
+      left: 0;
+      right: 0;
+      top: 0;
+      right: 0;
+      height: 100%;
+      width: 100%;
+      background: rgba(0,0,0,0.5);
+      z-index: 499;
```

Comment:
`499`? :D

Reply:
this feels like a magic number

Reply:
😂 The menu uses 500, so I chose a lower number. Do we have best practices around this?

Reply:
No, I don't think we do... I was just curious :)

Reply:
We should probably audit the project, maybe during cooldown to see if in fact all these `z-index`es are required and if so, replace them with more sensible ones using the utility classes, e.g. `z-10`.